src/containerapp/azext_containerapp/_params.py

```diff
@@ -161,6 +161,10 @@ def load_arguments(self, _):
         c.argument('environment_name', options_list=['--environment'], help="The environment name.")
         c.argument('resource_group_name', arg_type=resource_group_name_type, id_part=None)
 
+    with self.argument_context('containerapp env') as c:
+        c.argument('public_network_access', arg_type=get_enum_type(['Enabled', 'Disabled']),
+               help="Allow or block all public traffic", is_preview=True)
```

Comment:
ERROR: /mnt/vss/_work/1/s/src/containerapp/azext_containerapp/_params.py:166:16: E128 continuation line under-indented for visual indent